queries.sql

```diff
@@ -21,5 +21,83 @@ SELECT * FROM animals WHERE neutered = true;
 -- Find all animals not named Gabumon.
 SELECT * FROM animals WHERE NOT name = 'Gabumon';
 
--- Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)
+/* 
+** Find all animals with a weight between 10.4kg and 17.3kg (including the 
+** animals with the weights that equals precisely 10.4kg or 17.3kg)
+*/
 SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+-- Start a transaction to set all species to 'unspecified' and roll it back afterwards
+BEGIN;
+UPDATE animals SET species='unspecified';
+SELECT * FROM animals;
+ROLLBACK;
+
```

Comment:
- [ ] You should verify the changes successfully rollbacked.

> Inside a transaction update the animals table by setting the species column to unspecified. Verify that change was made. Then roll back the change and **verify that species columns went back to the state before transaction**.

```suggestion
BEGIN;
UPDATE animals SET species='unspecified';
SELECT * FROM animals;
ROLLBACK;
SELECT species from animals; -- verify that change was undone


```

Reply:
Thank you for pointing that out. I greatly appreciate it.

Reply:
This change has been made succesfully